src/ActionSheet/index.tsx

```diff
@@ -0,0 +1,275 @@
+import * as React from 'react';
+import {
+  Animated,
+  BackHandler,
+  Easing,
+  StyleSheet,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+import ActionGroup from './ActionGroup';
+import { ActionSheetOptions } from '../types';
+
+type ActionSheetState = {
```

Comment:
unless there's something specific to keeping the name ActionSheetState and ActionSheetProps can you call them State and Props? Still learning typescript ways 😅

Reply:
👍